### DISCUSSION TITLE
How connect 2 fedora with gui?

### DISCUSSION THREAD

**Discussion by simpsonetti** [Ylianst/MeshCentral#6304] [Q&A]
Hi there, 
i've tested here Windows and Mac and this works very well with MC. 
But now, there is a Fedora System here, which has to be connect to. 
So how can i connect to a gui ( by default gnome) in order to use this maschine remote?
Regards Sebastian

**Comment by si458** [comment]
for starters, you must switch the display system from wayland to x11 otherwise the screen will be blank
https://github.com/Ylianst/MeshAgent?tab=readme-ov-file#special-note-about-kvm-support-on-linux
then just simply copy+paste the command script in the meshcentral ui under add agent and select linux

**Reply by simpsonetti** [reply]
Yes, i got it, the default is "gnome" but if i change to "gnome X11" it works. 
Thanks.

**Reply by si458** [reply]
yep thats it! so use the X11 option, but you can also use the link above to CHANGE a file to always use X11 by default
